cli/internal/plugins/versions.go

```diff
@@ -58,14 +62,11 @@ func getLatestCQPluginRelease(ctx context.Context, name string, typ PluginType)
 	if err != nil {
 		return "", fmt.Errorf("unmarshaling manifest response: %w", err)
 	}
-	return extractVersionFromTag(mr.Latest), nil
+	return extractPluginVersionFromTag(mr.Latest, name, typ), nil
 }
 
-// extractVersionFromTag takes a tag of the form "plugins-source-test-v0.1.21" and returns
-// the version, i.e. "v0.1.21"
-func extractVersionFromTag(tag string) string {
-	parts := strings.Split(tag, "-")
-	return parts[len(parts)-1]
+func extractPluginVersionFromTag(tag, name string, typ PluginType) string {
```

Comment:
I don't really think this function is needed and we can inline that but just nit.